### DISCUSSION TITLE
Reverse Geocoding - GeoTIFF to Radar Coordinates?

### DISCUSSION THREAD

**Discussion by rbiessel** [isce-framework/isce2#396] [Q&A]
Hello All,

Is there a straightforward way, using ISCE or GDAL, to project a geocoded raster back into radar coordinates given the lat.rdr and lon.rdr virtual rasters? Using TOPSstack, I coregistered a stack of SLCs and I want to incorporate an external GeoTIFF into my next workflow and I only want to geocode the final outputs. ISCE does this for the DEM. Is there a general way to access this kind of routine for different rasters?

**Comment by songzwgithub** [comment]
Hello, I wanted to ask if you have found a way to convert geographic coordinates to radar coordinates?
